test/spec/services/solrSettingsSvc.js

```diff
@@ -26,52 +26,52 @@ describe('Service: solrSettingsSvc', function () {
     var settings = stubSettings();
     var startUrl = 'http://localhost:8983/solr/example?q=*:*&fl=title catch_line';
     solrSettingsSvc.fromStartUrl(startUrl, settings);
-    expect(settings.searchUrl).toEqual('http://localhost:8983/solr/example');    
+    expect(settings.searchUrl).toEqual('http://localhost:8983/solr/example');
     expect(settings.fieldSpecStr).toEqual('title catch_line');
     expect(settings.searchArgsStr).toEqual('q=*:*');
     expect(settings.whichEngine).toEqual(0);
     expect(settings.startUrl).toEqual('http://localhost:8983/solr/example?q=*:*&fl=title catch_line');
   });
-  
+
   it('uses default (*) fieldspec when no fl specified', function() {
     var settings = stubSettings();
     var startUrl = 'http://localhost:8983/solr/example?q=*:*';
     solrSettingsSvc.fromStartUrl(startUrl, settings);
-    expect(settings.searchUrl).toEqual('http://localhost:8983/solr/example');    
+    expect(settings.searchUrl).toEqual('http://localhost:8983/solr/example');
     expect(settings.fieldSpecStr).toEqual('*');
     expect(settings.searchArgsStr).toEqual('q=*:*');
     expect(settings.whichEngine).toEqual(0);
     expect(settings.startUrl).toEqual('http://localhost:8983/solr/example?q=*:*');
   });
-  
+
   it('uses start URL even with no args', function() {
     var settings = stubSettings();
     var startUrl = 'http://localhost:8983/solr/example';
     solrSettingsSvc.fromStartUrl(startUrl, settings);
-    expect(settings.searchUrl).toEqual('http://localhost:8983/solr/example');    
+    expect(settings.searchUrl).toEqual('http://localhost:8983/solr/example');
     expect(settings.fieldSpecStr).toEqual('*');
     expect(settings.searchArgsStr).toEqual('q=*:*');
     expect(settings.whichEngine).toEqual(0);
     expect(settings.startUrl).toEqual('http://localhost:8983/solr/example');
   });
-  
+
   it('updates start URL from args updates', function() {
     var settings = stubSettings();
     var startUrl = 'http://localhost:8983/solr/example?q=*:*&fl=title catch_line';
     solrSettingsSvc.fromStartUrl(startUrl, settings);
 
-    settings.searchArgsStr = 'q=*:*\n&fq=blah';
+    settings.searchArgsStrShow = 'q=*:*\nfq=blah';
```

Comment:
It would be good to have tests that
1. have just a newline (what this test is now)
2. have just ampersands
3. have both